default.json

```diff
@@ -7,7 +7,7 @@
       "customType": "regex",
       "fileMatch": [".*"],
       "matchStrings": [
-        "renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>\\S+?)(?: (packageName)=(?<packageName>\\S+?))?(?: versioning=(?<versioning>\\S+?))?(?: registryUrl=(?<registryUrl>\\S+?))?\\n\\s*(?:\\S+?_)?(?:VERSION|version|TAG|tag)\\s*[?]?[=:]\\s*\"?(?<currentValue>\\S+?)\"?\\s"
+        "renovate:\\s*datasource=(?<datasource>\\S+)\\s+depName=(?<depName>\\S+)(?:\\s+(package|lookup)Name=(?<packageName>\\S+))?(?:\\s+versioning=(?<versioning>\\S+))?(?:\\s+registryUrl=(?<registryUrl>\\S+))?\\n.+?_(?:VERSION|version|TAG|tag)\\s*[?=:]+\\s*\"?(?<currentValue>[^\" \\t\\n]+)\"?"

```

Comment:
```suggestion
        "renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>\S+?)(?: (packageName)=(?<packageName>\S+?))?(?: versioning=(?<versioning>\S+?))?(?: registryUrl=(?<registryUrl>\S+?))?\n.*(?:\S+?_)?(?:VERSION|version|TAG|tag)\s*[?]?[=:]\s*"?(?<currentValue>[^"\s]+)"?"
```

Reply:
Maybe we should just remove `(?:\\S+?_)?` altogether? So only `\n.*(?:VERSION.....)`, i.e.:

```yaml
# renovate: datasource=some-source depName=some-name
VERSION: v1

# renovate: datasource=some-source depName=some-name
MY_VERSION: v1
```

```java
// renovate: datasource=some-source depName=some-name
static final String CONFLUENT_VERSION = "7.5.1";
```

What do you think, @sverrehu ?